### PR TITLE
chore: fix pageHeader dependency [DET-7721]

### PR DESCRIPTION
**Pull request by hamidzr**
## Description

- revert sharing pageHeader
- update the shared-module tester for git subtree

## Test Plan

`cd webui/react/src/shared; ./bin/test.py --repos saas --test-local`

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: webui/react/src/shared/components/Page/Page.tsx [hunk 1]
```diff
@@ -36,15 +36,7 @@ const Page: React.FC<Props> = (props: Props) => {
...
-      {showHeader && (
-        <PageHeader
-          breadcrumb={props.breadcrumb}
-          options={props.options}
-          sticky={props.stickyHeader}
-          subTitle={props.subTitle}
-          title={props.title}
-        />
-      )}
+      {showHeader && props.pageHeader && props.pageHeader}
```
Comment (johnkim-det) [inline webui/react/src/shared/components/Page/Page.tsx:39]: redundant `&& props.pageHeader`?

**Comment by trentwatt** [comment]
test plan command fails, not sure what to do with this for now reassigning

**Comment by hamidzr** [comment]
https://determined-ai.slack.com/archives/C03H5KZPU30/p1656549096513339 related discussion.

closing as this has been added through https://github.com/determined-ai/determined/pull/4430

**Comment by hamidzr** [comment]
> test plan command fails, not sure what to do with this for now.
> 
> reassigning

will work on fixing the test failure. I think the failure is due to a new change that was introduced in shared-web between when the pr was prepared and reviewed

**Pull request closed without merging**